QWeb/__main__.py

```diff
@@ -29,12 +29,25 @@ def cli():
         '%(prog)s --list Get (lists all keywords startin with "Get")\n'
         '%(prog)s --show TypeText (displays documentation for keyword "TypeText")',
         description='Note: This module is meant to be used as a robot famework library.'
-        'Command line interface only provides access to keyword documentation.')
```

Comment:
I wonder why the formatter changed these? Maybe different line length setting or something. For me looks uglier when divided to (unnecessary) multiple lines

Reply:
Probably due to line length.
I think I need to check yapf line len setting and rerun the formatter...

Reply:
Setting line len to 100 with yapf mostly fixed this issue